### PR TITLE
fix: properly pass generated logback to gatling tasks

### PR DESCRIPTION
**Pull request by eshepelyuk**
resolves gatling/gatling#3948

### CODE REVIEW

File: src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy [hunk 1]
```diff
@@ -51,6 +51,8 @@ class GatlingRunTask extends DefaultTask implements JvmConfigurable {
     void gatlingRun() {
         def gatlingExt = project.extensions.getByType(GatlingPluginExtension)
 
+        def logbackFile = new File(project.buildDir, "generated/gatlingLogback/logback.xml")
```
Comment (slandelle) [inline src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy:54]: * Duplicated code, file should be resolved with `LogbackConfigTask#logbackFile`
* Maybe move resolution close to usage, line 64

**Review by slandelle — CHANGES_REQUESTED** [review]

File: src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy (hunk 1 above)
Comment (eshepelyuk) [reply src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy]: Done

**Review by Isammoc — APPROVED** [review]
LGTM

File: src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy [hunk 2]
```diff
@@ -51,6 +51,8 @@ class GatlingRunTask extends DefaultTask implements JvmConfigurable {
     void gatlingRun() {
         def gatlingExt = project.extensions.getByType(GatlingPluginExtension)
 
+
+
```
Comment (slandelle) [inline src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy:55]: nit: please remove extra empty lines

**Review by slandelle — CHANGES_REQUESTED** [review]
One last nit and then LGTM.
Thanks!

File: src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy (hunk 2 above)
Comment (eshepelyuk) [reply src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy]: pushed

**Review by slandelle — APPROVED** [review]

**Pull request merged**